src/action.ts

```diff
@@ -94,6 +112,36 @@ async function handleIssueEdited(options: IssueEditedOptions) {
     page_id: pageId,
     properties: parsePropertiesFromPayload(payload),
   });
+
+  const bodyBlocks = getBodyChildrenBlocks(payload);
+
+  const existingList = (
+    await notion.client.blocks.children.list({
+      block_id: pageId,
+    })
+  ).results;
+
+  const overlap = Math.min(bodyBlocks.length, existingList.length);
+
+  await Promise.all(
+    bodyBlocks.slice(0, overlap).map((block, index) =>
+      notion.client.blocks.update({
+        block_id: existingList[index].id,
+        ...block,
+      })
+    )
+  );
+
+  if (bodyBlocks.length > existingList.length) {
+    await notion.client.blocks.children.append({
+      block_id: pageId,
+      children: bodyBlocks.slice(overlap),
+    });
+  } else if (bodyBlocks.length < existingList.length) {
+    await Promise.all(
+      existingList.slice(overlap).map(block => notion.client.blocks.delete({block_id: block.id}))
```

Comment:
Similar to the comment above, does anything in the api let us delete all blocks in a page in one call?